### ISSUE TITLE
Field separator does not working at version 0.92 when using mysql

### ISSUE THREAD

**Issue by calebeaires** [xo/usql#278]
**go version** go1.16.6 linux/amd64
**usql** - 0.92
**mysql**  Ver 8.0.25-0ubuntu0.20.04.1 for Linux on x86_64

Trying to use field separator on USQL cli but I had no success. 

```bash
usql --csv --field-separator="|"  my://connection_string_here  -w  --command "SELECT name, email from users limit 2"  > result.csv
```

The result printed at file is separated by `'`, wich is extrange.

The result

```
name'email
TI Labs'contact@email.com
Waldo Garrones'waldo@email.com
```

The expected result is a txt with `|`  since I am using the --field-separator="|" at the command line. How to fix that?

**Comment by kenshaw** [comment]
It seems to be working for me:

```sh
$ ./build/linux/0.9.2/usql  --csv --field-separator="_" -c 'select * from books' my://booktest:booktest@localhost/booktest
Connected with driver mysql (10.6.3-MariaDB-1:10.6.3+maria~focal)
"book_id"_"author_id"_isbn_"book_type"_title_year_available_description_tags
1_1_1_FICTION_"my book title"_2016_2021-07-21T20:41:15+07:00__
2_1_2_FICTION_"changed second title"_2016_2021-07-21T20:41:15+07:00__"cool disastor"
3_1_3_FICTION_"the third book"_2001_2021-07-21T20:41:15+07:00__cool

$ ./build/linux/0.9.2/usql  --csv --field-separator="|" -w -c 'select * from books' my://booktest:booktest@localhost/booktest
Connected with driver mysql (10.6.3-MariaDB-1:10.6.3+maria~focal)
book_id|author_id|isbn|book_type|title|year|available|description|tags
1|1|1|FICTION|"my book title"|2016|2021-07-21T20:41:15+07:00||
2|1|2|FICTION|"changed second title"|2016|2021-07-21T20:41:15+07:00||"cool disastor"
3|1|3|FICTION|"the third book"|2001|2021-07-21T20:41:15+07:00||cool
```

Considering that it would appear `usql` is using the `'` as a separator, I'd suggest that there is an unescaped character somewhere in the script being used.

**Comment by calebeaires** [comment]
For some reason, the `\f` was set to `'` than even if using --field-separator property the csv outups the values with that other char. Here is what I've done:

First: enter at usql (without drive settings)
At bash
```bash
usql
```

Second set \f with the value you want
At usql command line

```bash
\f "|"
```

Now futures bash usql calls will be using the correct one.

**Comment by kenshaw** [comment]
Perhaps check your `.usqlrc`?

**Comment by calebeaires** [comment]
I did and it was not there. 

I even created one when there is no one at my server. Here is what happen...

After set `\f "|"` at `.usqlrc` everything goes well but when using the command line at bash and every time I run the command to save  **output** on csv, the file gets dirty with this text:  Field separator is "|". (I used --set SHOW_HOST_INFORMATION=false, but it does not hide the command either)
